ufs_err_hist.h

```diff
@@ -0,0 +1,16 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+/* Copyright (C) 2019 Western Digital Corporation or its affiliates */
+
+#ifndef UFS_ERR_HIST_H_
+#define UFS_ERR_HIST_H_
+#include "options.h"
+
+enum vendor_cmd_type {
```

Comment:
I think this enum is redundant

Reply:
I think, There's no need to be too particular.  if there is no big issue, cause disaster. it is ok.
The key and primary mission is to enable all features and make it available to use for everybody.
this is just my own opinion.  
Of course, under the premise that we can guarantee problem-free and bug-free. if we can improve its nice coding style, it is better.

Reply:
Correct the enum is redundant and was removed. Please continue review